src/rules/variableNameRule.ts

```diff
@@ -64,11 +87,19 @@ class VariableNameWalker extends Lint.RuleWalker {
         }
     }
 
-    private handleVariableName(name: ts.Identifier) {
+    private handleVariableNameFormat(name: ts.Identifier) {
```

Comment:
Not possible w/o adding another parameter to the method. (Both methods aren't always called together)